src/NextGen/DonationForm/Controllers/DonateController.php

```diff
@@ -40,8 +39,9 @@ public function donate(DonateControllerData $formData, PaymentGateway $registere
 
         $this->saveCustomFields($form, $donation, $formData->getCustomFields());
 
-        // setting sessions is required for legacy receipts
-        $this->setSession($donation, $donor);
+        $this->filterSuccessPageUri($formData->getDonationConfirmationReceiptViewRouteUrl($donation));
```

Comment:
I would consider a different name here, since in the context of WordPress "filter" insinuates a return value that may or may not be the same as the argument.

Reply:
resolved b55ef555